indico/web/templates/_session_menu.html

```diff
@@ -0,0 +1,220 @@
+{% macro _settings_section(section_id, heading_id, title, heading_extra='') %}
+    <section id="{{ section_id }}" aria-labelledby="{{ heading_id }}">
+        <heading>
+            <h2 id="{{ heading_id }}">{{ title }}</h2>
+            {{ heading_extra }}
+        </heading>
+        {{ caller() }}
+    </section>
+{% endmacro %}
+
+{% macro _render_protection_info(protection_data) %}
+    {% set title %}{% trans %}Page access information{% endtrans %}{% endset %}
+    {% call _settings_section('protection-details-info', 'protection-details', title) %}
+        <div data-mode="{{ protection_data.protection_mode }}">
+            <div>
+                <p>
+                    {% if protection_data.protection_mode == 'public' %}
+                        {% trans %}The information on this page is visible to anyone{% endtrans %}
+                    {% elif protection_data.protection_mode == 'network' %}
+                        {% trans count=protection_data.networks_with_access|length, networks=protection_data.networks_with_access|join('/') %}
+                            The information on this page is restricted for display on the {{ networks }} network only.
+                            {% pluralize %}
+                            The information on this page is restricted for display on the {{ networks }} networks only.
+                        {% endtrans %}
+                    {% elif protection_data.protection_mode == 'restricted' %}
+                        {% trans %}
+                            The information on this page is restricted for display to named individuals or specific groups.
+                        {% endtrans %}
+                    {% endif %}
+                </p>
+                {% if protection_data.disclaimer %}
+                    <p class="protection-disclaimer">
+                        {{ protection_data.disclaimer|sanitize_html|trim }}
+                    </p>
+                {% endif %}
+            </div>
+        </div>
+    {% endcall %}
+{% endmacro %}
+
+{% macro _render_timezone_selector(timezone_data) %}
+    {% set title %}{% trans %}Timezone{% endtrans %}{% endset %}
+    {% call _settings_section('tz-selector-controls', 'tz-selector', title) %}
+        {% if timezone_data.has_no_choice %}
+            <p class="tz-locked-note">{% trans active_tz=timezone_data.active_tz %}The active timezone is {{ active_tz }} and it cannot be changed{% endtrans %}.</p>
```

Comment:
Maybe this can be worded a bit more nicely?

"Management areas always use the timezone of the event/category" or something like that...

Reply:
Is that the only place where timezone is locked?